src/platforms/python/common/performance/index.mdx

```diff
@@ -48,12 +48,12 @@ Alternatively, instead of adding to the top-level transaction, you can make a ch
 You can choose the value of `op` and `description`.
 
 ```python
-from sentry_sdk import Hub
+import sentry_sdk
 
 def process_item(item):
     transaction = Hub.current.scope.transaction
     # omitted code...
-    with transaction.start_child(op="http", description="GET /") as span:
+    with sentry_sdk.start_span(op="http", description="GET /") as span:
```

Comment:
@untitaker could you describe what is the intended behavior?
What does starting a span without a reference to a transaction do?

Reply:
I can add this to the document but basically `start_span` would return a noop context manager. This is not what is actually happening under the hood: In practice a span is constructed and destroyed on the scope, but never put into any sort of array or other datastructure to be put into a transaction event (I'd leave that up to implementation)

Reply:
👍 

Okay, if I got you right, it would have the side-effect of changing the scope of the current hub.
This is only useful if at times there would also be a transaction in the same scope, in which case the span would not be discarded, but instead would become part of the ongoing transaction -- is that accurate?

Reply:
It's useful to have this noop in case there is no wrapping transaction. The otehr two alternatives I see is:

* throw error that there is no transaction -- functions would have to guard against this at all times
* start own transaction -- if there are 100 db spans in an endpoint and the endpoint for some reason did not start a transaction, this can lead to performance and billing problems